spring-boot-project/spring-boot-docs/src/docs/asciidoc/actuator/metrics.adoc

```diff
@@ -259,12 +287,14 @@ The location of the Elastic server to use can be provided using the following pr
 	      elastic:
 	        host: "https://elastic.example.com:8086"
 ----
+====
 
 [[actuator.metrics.export.ganglia]]
 ==== Ganglia
 By default, metrics are exported to {micrometer-registry-docs}/ganglia[Ganglia] running on your local machine.
-The http://ganglia.sourceforge.net[Ganglia server] host and port to use can be provided using:
+You can provide the http://ganglia.sourceforge.net[Ganglia server] host and port to use by using:
```

Comment:
"use by using" reads a little bit strangely to me. How about "You can configure the Ganglia server host and port to use, as the following example shows:" instead?

Reply:
Yup. I created an awkwardness. Thanks.